applications/annotate_points.md

```diff
@@ -0,0 +1,314 @@
+
+```python
+@viewer.bind_key(',')
+def prev_label(event):
+    """Keybinding to decrement to the previous label with wraparound"""
+    current_properties = points_layer.current_properties
+    current_label = current_properties['label'][0]
+    ind = list(labels).index(current_label)
+    n_labels = len(labels)
+    new_ind = ((ind - 1) + n_labels) % n_labels
+    new_label = labels[new_ind]
+    current_properties['label'] = np.array([new_label])
+    points_layer.current_properties = current_properties
+```
+
+## Mousebinding to iterate through labels
+Similar to keybindings, we can also bind functions to mouse events such as clicking or dragging. Here, we create a function that will increment the label after a point is added (i.e., the mouse is clicked in the viewer canvas when in the point adding mode). This is convenient for quickly adding all labels to a frame, as one can simply click each feature in order without having to manually swap labels. To achieve this, we first check if the points layer is the the adding mode (`layer.mode == 'add'`). If so, we then reuse the next_label() function we defined above in the keybindings to increment the label. Finally, 
+
+```python
+def next_on_click(layer, event):
+    """Mouse click binding to advance the label when a point is added"""
+    # only do something if we are adding points
+    if layer.mode == 'add':
+        next_label()
+
+        # by default, napari selects the point that was just added
+        # disable that behavior, as the highlight gets in the way
+        layer.selected_data = []
+```
+
+After creating the function, we then add it to the `points_layer` mouse drag callbacks. In napari, clicking and dragging events are both handled under the `mouse_drag_callbacks`. For more details on how mouse event callbacks work see the examples [[1](https://github.com/napari/napari/blob/master/examples/custom_mouse_functions.py), [2](https://github.com/napari/napari/blob/master/examples/mouse_drag_callback.py)].
+
+```python
+# bind the callback to the mouse drag event
+points_layer.mouse_drag_callbacks.append(next_on_click)
+```
+
+## Using the GUI
+### Launching the GUI
+Now that you've put it all together, you should be ready to test! You can call the function as shown below.
+
+```python
+im_path = '<path to directory with data>/*.png'
+output = '<path to directory with data>/annotations.csv'
+point_annotator(im_path, labels=['ear_l', 'ear_r', 'tail'], output_path=output)
+
+```
+
+### Saving the annotations
+Once we are happy with the annotations, we can save them to a CSV file using the builing CSV writer for the points layer. To do so, first, select the "Points" layer in the layer list and then click "Save Selected layer(s)"  in the "File" menu or press control+S (cmd+S on Mac OS)  to bring up the file save dialog. From here you can enter the file path and save the annotation coordinates as a CSV.
+
+![image]({{ '/assets/tutorials/points_save_dialog.png' | relative_url }})
```

Comment:
We can also use `points_layer.save('path/to/file.csv')`, which I think is worth pointing out!

Reply:
I can add some text mentioning the `points_layer.save()` method.